src/main/java/net/sf/javaanpr/imageanalysis/PeakComparator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2013 JavaANPR contributors
+ * Copyright 2006 Ondrej Martinsky
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package net.sf.javaanpr.imageanalysis;
+
+import java.util.Comparator;
+
+public class PeakComparator implements Comparator<Peak> {
+    private PlateVerticalGraph graphHandle = null;
+
+    public PeakComparator(PlateVerticalGraph graph) {
```

Comment:
Let's only pass yValues into this comparator. It will be easier to test and more encapsulated.